lang/cs/Org.Apache.REEF.Client/YARN/IApplicationReport.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Ah just realized Application is internal. Perhaps, creating IApplication and let Application implement IApplication will make it simpler